webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  mode: 'production',
+  devtool: 'source-map',
+  entry: {
+    application: './app/javascript/application.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js)$/,
+        exclude: /node_modules/,
+        use: ['babel-loader'],
+      },
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/233621388-4c1d5352-2f81-4471-bd99-c1c7ceb23362.png)

- [ ] Great work on implementing some of the requested changes :+1: but I guess while making the changes you broke your app because when I run your app with ./bin/dev command then I saw the error in the terminal which you can see in the image above :thinking: and your frontend part is also not working because of it :fearful: Kindly fix this issue to make your work as expected :+1: